### PR TITLE
tests: addition of test_examples_app.py

### LINKED ISSUE #55: tests: addition of test_example_app.py
The package contains `examples/app.py` example application, but there is no test
for it in `tests/test_example_app.py`.  It should be added.  See existing examples:

- https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py
- https://github.com/inveniosoftware/invenio-marc21/blob/master/tests/test_example_app.py

### PR DESCRIPTION
**Pull request by hachreak**
* Adds test_examples_app.py. (closes #55)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
@jirikuncar :shipit: ?

File: tests/test_examples_app.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Test example app."""
+
+import json
```
Comment (jirikuncar) [inline tests/test_examples_app.py]: unused?

Comment (hachreak) [inline tests/test_examples_app.py]: done

**Comment by hachreak** [comment]
ping @jirikuncar

**Pull request merged**